app/index.js

```diff
@@ -58,10 +58,9 @@ var FluxGenerator = yeoman.generators.Base.extend({
     this.mkdir('src/js/stores');
     this.mkdir('src/js/constants');
     this.mkdir('src/js/actions');
-    this.copy('js/index.js', 'src/js/index.js');
+    this.copy('js/index.jsx', 'src/js/index.jsx');
     this.copy('_index.html', 'src/index.html');
     this.copy('js/constants.js', 'src/js/constants/AppConstants.js');
-    this.copy('js/components/app.js', 'src/js/components/App.js');
```

Comment:
I definitely prefer to have the generator create a single `App` component, since these [`controller-views`](http://facebook.github.io/flux/docs/overview.html#views-and-controller-views) hold a special place in Flux. This would need to stay before merging...